### ISSUE TITLE
lang=it @ owasp.org

### ISSUE THREAD

**Issue by drwetter** [OWASP/Top10#621]
che c...? ;-) 
 
![image](https://user-images.githubusercontent.com/8036727/134778100-e7405b00-47a5-49f6-80a5-f94b093477b7.png)

**Comment by sslHello** [comment]
Is somebody working on the Italian version?
Ciao Torsten

**Comment by elbae** [comment]
Can we not translate categories name in italian please?

**Comment by ricsirigu** [comment]
> Can we not translate categories name in italian please?

O almeno non tradotto letteralmente :(

**Comment by sslHello** [comment]
Yes we do have committet the first PR for the Italien version (#633).
We'll close this issie when pages will be rebuilt from master branch.
Cheers Torsten

**Comment by vanderaj** [comment]
This was merged. Closing
